nailgun/entities.py

```diff
@@ -4036,6 +4037,70 @@ def errata(self, synchronous=True, **kwargs):
         response = client.get(self.path('errata'), **kwargs)
         return _handle_response(response, self._server_config, synchronous)
 
+    def traces(self, synchronous=True, **kwargs):
+        """List services that need restarting on the host
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all content decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.get(self.path('traces'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous)
+
+    def bulk_traces(self, synchronous=True, **kwargs):
+        """List services that need restarting on the specified set of hosts
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all content decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.post(self.path('bulk/traces'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous)
+
+    def resolve_traces(self, synchronous=True, **kwargs):
+        """Resolve traces the host
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all content decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.put(self.path('traces/resolve'), **kwargs)
```

Comment:
The Path is incorrect , this wont work , See my another comment.

Reply:
It is not for the most recent Katello version:
https://github.com/Katello/katello/pull/8945/files#diff-e2385b3bc0001ddfb05e639f7ff3278dR17

Reply:
Marking this comment as resolved since https://<any-sat-6.8-snap17-or-newer>//apidoc isnt showing the correct path but the actual path is working with `host_id`.